sections/main-product.liquid

```diff
@@ -496,6 +496,9 @@
       "@type": "Thing",
       "name": {{ product.vendor | json }}
     },
+    {%- if product.standard_product_type -%}
+      "category": {{ product.standard_product_type.type_path | json }},
+    {%- endif -%}
```

Comment:
If there is a "custom type" shoud we:
- Keep using `standard_product_type`? 
- Use `custom_product_type` instead?
- Set category to `null`?

Reply:
Good question. I think if custom is set, we should keep using `standard_product_type`, which is hierarchical and matches the expectations of schema.org's `category`. I could be wrong, but I don't see much value in injecting `custom_product_type` in here when it could be used in so many merchant-specific ways, and may not follow the format (which specifies using `>` or `/` to build the path).